src/index.css

```diff
@@ -26,6 +26,17 @@ body {
   cursor: pointer;
 }
 
+.overlay {
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  z-index: -10;
+  background-image: url(./assets/img/bg.jpg);
+  background-position: top;
+  background-size: cover;
+  filter: blur(5px);
+}
+
```

Comment:
don't change the structure, leave this block of code after the body

Reply:
I changed it in the `index.js` file, you should check it out

Reply:
Ok, I forget to pull the changes!